recipes/libignition-transport/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=True
+
+make -j${CPU_COUNT}
```

Comment:
I think you have to run `ctest` after building to run the tests.

Reply:
Done. I added `ctest` to invoke the test passes.